Anchor.toml

```diff
@@ -0,0 +1,16 @@
+[features]
+seeds = false
+skip-lint = false
+
+[programs.localnet]
+{{project-name}} = "{{program-id}}"
+
+[registry]
+url = "https://api.apr.dev"
+
+[provider]
+cluster = "Localnet"
+wallet = "~/.config/solana/id.json"
+
+[scripts]
+test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/test.ts --exit"
```

Comment:
in my template I added a build and test command which uses the light cli
any specific reason to do it otherwise in here?

Reply:
I will switch to using light CLI, my bad.

Reply:
Nope, in your template (in light-anchor) you're also using `yarn run ts-mocha`:

https://github.com/Lightprotocol/anchor/blob/2b0af559f7c5d326f96ef196175635827ce762d3/cli/src/lib.rs#L800-L838

Exactly here:

https://github.com/Lightprotocol/anchor/blob/2b0af559f7c5d326f96ef196175635827ce762d3/cli/src/lib.rs#L817

So I didn't change Anchor.toml at all.

Reply:
ah we don`t use this we just use package.json